Documentation/ApiOverview/CachingFramework/Configuration/Index.rst

```diff
@@ -43,26 +43,9 @@ Administrators can overwrite specific settings of the cache configuration in
 :file:`config/system/settings.php`. Here is an example configuration to switch
 **pages** to the **redis** backend using database 3:
 
-..  code-block:: php
-    :caption: config/system/settings.php
-
-    return [
-        // ... other configuration
-
-        'SYS' => [
-            'caching' => [
-                'cacheConfigurations' => [
-                    'pages' => [
-                        'backend' => \TYPO3\CMS\Core\Cache\Backend\RedisBackend::class,
-                        'options' => [
-                            'database' => 3,
-                        ],
-                   ],
-                ],
-            ],
-        ],
-    ];
-
+..  literalinclude:: _settings.php
+    :language: php
+    :caption: config/system/settings.php | typo3conf/system/settings.php
```

Comment:
As the settings.php is thought to be used with the Install Tool GUI, while the additional.php is used for manual changes I wonder, if we should rather use the additional.php for such examples? also there  such examples could be written in fewer lines and without leaving out parts